assests/javascript/index.js

```diff
@@ -1 +1,225 @@
-//
+const apiKey = "8cf248a4fd398e2908cf52a9f187375b";
+const mainSection = document.getElementById("main");
+
+let city = {};
+let curentWeather = {};
+let dailyForecastingWeatherList = {};
+
+getWeather();
+
+async function getWeather() {
+  await getCity();
+  await getCurrentWeather(city);
+  await getForecastingWeather(city);
+
+  showWeather();
+
+  if (city || curentWeather || dailyForecastingWeatherList === {}) {
+    showErrorMassage();
+  }
```

Comment:
`dailyForecastingWeatherList === {}` this will be false.
![image](https://user-images.githubusercontent.com/1394580/197611483-cafbbb5b-b364-4507-a4c9-056b7a52dd68.png)

Reply:
wow